example/ios/example.xcodeproj/project.pbxproj

```diff
@@ -138,7 +138,8 @@
 			children = (
 				84EC058429936A3A001904E4 /* Identity.xcframework */,
 			);
-			path = Frameworks;
+			name = Frameworks;
+			path = ../../Identity.xcframework;
```

Comment:
When you went to build in Xcode were you having an issue with the Framework not showing up?

Reply:
Yes :/

Reply:
Weird. Thanks for fixing.